### PR TITLE
adds benchmarking tutorial

### LINKED ISSUE #448: Add benchmarking capabilities
One member approached me during PyCon asking about benchmarking capabilities. During the discussion, they mentioned they want to improve query performance and they wanted the ability to run tests across different databases:

Example:

1. SQL server on prem vs connecting via JupySQL
2. Testing query speed on different database access points

Others in the group asked if it was possible to include a mechanism to track how long it takes to complete a query.

### PR DESCRIPTION
**Pull request by AnirudhVIyer**
## Describe your changes
Added documentation to enable `jupyterlab-execute-time` plugin in `howto/benchmarking-time.md`. Added the profile in toc file.
## Issue number

Closes #448 

## Checklist before requesting a review

- [x] Performed a self-review of my code
- [x] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)
- [x] Added [tests](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#testing) (when necessary).
- [x] Added [docstring](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#documenting-changes-and-new-features) documentation and update the [changelog](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--537.org.readthedocs.build/en/537/

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204474565827721

### CODE REVIEW

**Comment by edublancas** [comment]
the PR is failing because of changes we did to master, please merge/rebase. that should fix it

**Comment by AnirudhVIyer** [comment]
Got it!

**Review by edublancas — CHANGES_REQUESTED** [review]
this is a good start, please adapt the tutorial so you show jupysql examples, you can use the duckdb guide as a reference: https://github.com/ploomber/jupysql/blob/master/doc/integrations/duckdb.md

**Comment by AnirudhVIyer** [comment]
Sure! will implement the changes.

**Comment by edublancas** [comment]
great! don't forget to re-request my review once the changes are done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # CHANGELOG
 
 ## 0.7.7dev
-
+* [Doc] Added Howto documentation for enabling JupyterLab cell run-time display ([#448](https://github.com/ploomber/jupysql/issues/448))
```
Comment (edublancas) [inline CHANGELOG.md:4]:
```suggestion
* [Doc] Added Howto documentation for enabling JupyterLab cell runtime display ([#448](https://github.com/ploomber/jupysql/issues/448))
```

File: doc/howto/benchmarking-time.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Display cell runtime in JupyterLab
+    keywords: jupyter, jupyterlab, sql
+    property=og:locale: en_US
+---
+
+# Benchmarking run-time in JupyterLab
```
Comment (edublancas) [inline doc/howto/benchmarking-time.md:20]:
```suggestion
# Benchmarking runtime
```

File: doc/howto/benchmarking-time.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+To record the time taken to run each cell
+in JupyterLab, we suggest using `jupyterlab-execute-time` 
+
+# Install jupyterlab-execute-time
```
Comment (edublancas) [inline doc/howto/benchmarking-time.md:24]: please change this and the other headings to be H2 headings, usually, we have an H1 and the top and then H2 sub-sections, sometimes even H3

```suggestion
## Installation
```

File: doc/howto/benchmarking-time.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+```sh
+pip install jupyterlab_execute_time
+```
+
+# Enable Record metadata Settings
```
Comment (edublancas) [inline doc/howto/benchmarking-time.md:30]: I think three headings for these is too much, you could structure like this

```
## Installation

## Usage

### (Optional) Store notebook metadata

### Sample screenshots

put your images here with some descriptive text



```

**Review by edublancas — CHANGES_REQUESTED** [review]

File: doc/howto/benchmarking-time.md (hunk 4 above)
Comment (AnirudhVIyer) [reply doc/howto/benchmarking-time.md:30]: Got it, will make the changes

**Review by edublancas — APPROVED** [review]

**Pull request merged**